tag-expressions/javascript/README.md

```diff
@@ -8,10 +8,9 @@
 ## Example
 
 ```js
-import {TagExpressionParser} from '@cucumber/tag-expressions'
-const parser = new TagExpressionParser()
+const tagExpressions = require("@cucumber/tag-expressions");
 
-const expressionNode = parser.parse('@tagA and @tagB')
+const expressionNode = tagExpressions.default('@tagA and @tagB');
```

Comment:
The fact that you have to use `.default` here tells me we should improve how the library is exported. It shouldn't be necessary. (It's not necessary in TypeScript, using `import`)

Rather than suggesting people use this workaround I'd like to fix the library.

Reply:
@aslakhellesoy I'm not much familiar with TypeScript syntax, but I surfed google and have done some fix. Kindly validate it.